### PR TITLE
LibM: Minor improvements for pow and long doubles

### PR DESCRIPTION
**Pull request by Hendiadyoin1**

### CODE REVIEW

File: Userland/Libraries/LibM/math.h [hunk 1]
```diff
@@ -29,6 +29,21 @@ __BEGIN_DECLS
 #define NAN __builtin_nan("")
 #define MAXFLOAT FLT_MAX
 
+// ruthlessly copied these from glibc
```
Comment (awesomekling) [inline Userland/Libraries/LibM/math.h:32]: This seems irrelevant.

Comment (Hendiadyoin1) [reply Userland/Libraries/LibM/math.h]: well than,
I removed that now
